data/harvard/law.free.cap.ala-app.17/226.1641532.json

```diff
@@ -0,0 +1,12 @@
+{
+    "citations": [
+        {
+            "type": "official",
+            "cite": "17 Ala. App. 226"
+        },
+        {
+            "type": "parallel",
+            "cite": "84 South. 415"
```

Comment:
Shouldn't this be `So.`?

Reply:
Thanks for the suggestion, I left it as it is in the pdf because it shows as South. 

Also based on [reporters.json](https://github.com/freelawproject/reporters-db/blob/main/reporters_db/data/reporters.json#L19239) "South." is mapped to "So." when is parsed with get_citations function from eyecite and then the corrected reporter becomes "So." 

Should I change it or leave it as shown in the pdf?

Reply:
Eh. It's all good. Merging.